recipe/meta.yaml

```diff
@@ -30,10 +30,16 @@ requirements:
   run:
     - python
```

Comment:
run should be:
```
run:
- python
- {{ pin_compatible('lz4-c', max_pin='x.x') }}
```

Because that matches what the run_exports of lz4-c would return.
sphinx and flake8 can be omitted because they are needed for development and building the doc, but not at runtime.